<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entradas</class>
 <widget class="QWidget" name="Entradas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compras</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

#Entradas{
background-color: rgb(199, 199, 199);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(199, 199, 199, 255), stop:1 rgba(170, 170, 170, 255));

}
QPushButton{
background-color: rgb(43, 132, 191);
color: rgb(255, 255, 255);
font-size: 15px;
border-radius: 13px;

}

QPushButton:hover{
	background-color: rgb(13, 92, 161);
}
QPushButton:pressed{
	
color: rgb(255, 255, 255);
	background-color: rgb(153, 193, 241);
font-size: 15px;
border-radius: 13px;
}
QGroupBox{
background-color:rgb(170, 170, 170);
	font: 700 9pt &quot;Segoe UI&quot;;
	font-size: 15px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>COMPRAS</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>130</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>130</height>
      </size>
     </property>
     <property name="title">
      <string>Filtros</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>88</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Período considerado:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>De:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="DateEdt_De">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Até:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="DateEdt_Ate">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 700 10pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>Notas Fiscais de Compras:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="Tview_Entradas">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="Btn_ConsultarDF">
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Atualizar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 700 10pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>Produtos Contidos na Nota Selecionada:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="Tview_ProdutosNota">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
